packages/design-system/tailwind.config.ts

```diff
@@ -1,23 +0,0 @@
-import type { Config } from 'tailwindcss';
-import borderRadius from './config/borderRadius';
```

Comment:
Is it really OK this whole file is deleted without any replacement?

Reply:
It was migrated to the .css file, the env variables there are the new @theme for Tailwind4

Reply:
Hmm…then we should remove the config files if they are useless.